src/embedded.js

```diff
@@ -325,12 +326,14 @@
             // - uxVersion                Integer. The version of the embedded user experience to display to signers (1 = legacy, 2 = responsive). This option is only honored if your account has accessed the API prior to Nov 14, 2015.
             // - requester                String. The email of the person issuing a signature request. Required for allowing 'Me + Others' requests
             // - whiteLabelingOptions     Object. An associative array to be used to customize the app's signer page
+            // - healthCheckInterval      Integer. The number of milliseconds to wait for a response from the iframe. If no response after that time the iframe will be closed.
```

Comment:
@freddyrangel Minor nitpick, but this is a "timeout" not an "interval". Also, it helps to have the units in the name, e.g., `healthCheckTimeoutMs`